newIDE/app/src/Utils/GDevelopServices/Play.js

```diff
@@ -121,14 +124,14 @@ export const listGameLeaderboards = async (
 
   const { uid: userId } = firebaseUser;
   const authorizationHeader = await getAuthorizationHeader();
-  return axios
-    .get(`${GDevelopPlayApi.baseUrl}/game/${gameId}/leaderboard`, {
+  const response = await axios.get(
+    `${GDevelopPlayApi.baseUrl}/game/${gameId}/leaderboard?deleted=false`,
```

Comment:
nipicking: maybe this function could be called `listActiveGameLeaderboards()` to be explicit that you will not receive the deleted ones if you call it

Reply:
Good idea!